package.json

```diff
@@ -263,7 +263,7 @@
     "regenerator-runtime": "0.13.7",
     "remark-parse": "9.0.0",
     "rosetta": "1.0.0",
-    "rxjs": "7.0.0",
+    "rxjs": "^7.8.0",
```

Comment:
We always pin our versions and looking at the changelog we should go to the latest available verison, 7.8.1